self_hosted/create_llvm_ir.jou

```diff
@@ -566,6 +569,8 @@ class AstToIR:
         elif ast->kind == AstStatementKind::Assert:
             condition = self->do_expression(&ast->expression)
             self->do_assert(condition)
+        elif ast->kind == AstStatementKind::Pass:
+            self->do_pass()
```

Comment:
Because we now have a `pass` keyword, we can just use it here:

```suggestion
            pass
```

You can then also delete the `do_pass()` method.